Pipfile

```diff
@@ -13,7 +13,7 @@ pytest-cov = "*"
 pylint = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.8"
```

Comment:
Is there a need to relock `Pipfile.lock`? Or do deps stay the same (I haven't tried it).